### PR TITLE
Provide sequence diagrams for Credentials API description.

### PR DESCRIPTION
**Pull request by sysexcontrol**
Signed-off-by: Karsten Frank <Karsten.Frank@bosch-si.com>

### CODE REVIEW

File: site/diagrams/sequenceDiagrams/connectToCredentials.txt [hunk 1]
```diff
@@ -0,0 +1,6 @@
+title Connection to Credentials API
```
Comment (sophokles73) [inline site/diagrams/sequenceDiagrams/connectToCredentials.txt]: `Connect to Credentials Service`?

File: site/diagrams/sequenceDiagrams/getCredentials_Success.txt [hunk 2]
```diff
@@ -0,0 +1,9 @@
+title Lookup Credentials (success)
```
Comment (sophokles73) [inline site/diagrams/sequenceDiagrams/getCredentials_Success.txt]: `Get Credentials (success)`?

File: site/diagrams/sequenceDiagrams/removeExistingCredentials_Success.txt [hunk 3]
```diff
@@ -0,0 +1,18 @@
+title Remove existing Credentials (success)
```
Comment (sophokles73) [inline site/diagrams/sequenceDiagrams/removeExistingCredentials_Success.txt]: `Remove Credentials (success)`? *existing* is implied by `(success)`, isn't it?

File: site/diagrams/sequenceDiagrams/updateExistingCredentials_Success.txt [hunk 4]
```diff
@@ -0,0 +1,9 @@
+title Update existing Credentials (success)
```
Comment (sophokles73) [inline site/diagrams/sequenceDiagrams/updateExistingCredentials_Success.txt]: `Update Credentials (success)`?

File: site/diagrams/sequenceDiagrams/connectToCredentials.txt [hunk 5]
```diff
@@ -0,0 +1,6 @@
...
+
+Client->Credentials: 1 open
```
Comment (sophokles73) [inline site/diagrams/sequenceDiagrams/connectToCredentials.txt]: missing:
````
Credentials->Client: 2 open
````

File: site/diagrams/sequenceDiagrams/connectToCredentials.txt [hunk 6]
```diff
@@ -0,0 +1,6 @@
...
+
+Client->Credentials: 2 attach("sender", "credentials/${tenant_id}")
```
Comment (sophokles73) [inline site/diagrams/sequenceDiagrams/connectToCredentials.txt]: missing:
`Credentials->Client: 4 attach("receiver", "credentials/${tenant_id}")`

File: site/diagrams/sequenceDiagrams/connectToCredentials.txt [hunk 7]
```diff
@@ -0,0 +1,6 @@
...
+Client->Credentials: 3 attach("receiver", "credentials/${tenant_id}/${reply-to}")
```
Comment (sophokles73) [inline site/diagrams/sequenceDiagrams/connectToCredentials.txt]: missing:
`Credentials->Client: 6 attach("sender", "credentials/${tenant_id}/${reply-to}")`

**Comment by sysexcontrol** [comment]
@sophokles73 : updated the diagrams and considered your comments. Can you have a look again, please?

File: site/diagrams/sequenceDiagrams/connectToCredentials.txt [hunk 8]
```diff
@@ -0,0 +1,6 @@
+title Connect to Credentials Service
+
+Client->Credentials: 2.1 attach("sender", "credentials/${tenant_id}")
+Credentials->Client: 2.2 attach("receiver", "credentials/${tenant_id}")
+Client->Credentials: 3.1 attach("receiver", "credentials/${tenant_id}/${reply-to}")
+Credentials->Client: 3.2 attach("sender", "credentials/${tenant_id}/${reply-to}")
```
Comment (sophokles73) [inline site/diagrams/sequenceDiagrams/connectToCredentials.txt]: same here

**Comment by sysexcontrol** [comment]
@sophokles73 : updated diagram.

**Comment by sysexcontrol** [comment]
@sophokles73 : now corrected, thx.

**Pull request merged**